xmodule/static/css-builtin-blocks/SequenceBlockDisplay.css

```diff
@@ -187,7 +209,7 @@ body.touch-based-device .xmodule_display.xmodule_SequenceBlock .sequence-nav ol
     display: none;
 }
 
-.xmodule_display.xmodule_SequenceBlock .sequence-nav-button {
+.xmodule_display.xmodule_SequenceBlock .sequence-nav-button, .dropdown-toggle {
```

Comment:
```suggestion
.xmodule_display.xmodule_SequenceBlock .sequence-nav-button {
```

This set's the max-width of the dropdown to 40px as well. This creates gaps in the border when the flex calculated width of the button is more than 40px. ~Instead we can apply the same styling as the unit buttons by adding it to the line 96.~

```diff
<removed>
```

Reply:
@xitij2000 I tested again to debug the mobile situation. I think adding it to the line 96, as I mentioned before isn't necessary at all.

However, I have found the issue with mobile screens, it enables veritical scrolling withing the top nav.
![image](https://github.com/user-attachments/assets/918070a7-40ed-43db-94b6-1df2f895f559)

In the master, this is similar to desktop, horizontal scrollable list. I am not sure what's the right fix would be. Rendering the dropdown only above mobile screen size could be an option.

Reply:
I don't think mobile is a use-case here. This UI is only used when rendering during an LTI launch AFAIK and I hope eventually it moves to the learning MFE. So unless someone is accessing this via another LMS in which it's embedded in mobile this won't show up.